website_media_size/static/src/css/website_media_size.css

```diff
@@ -0,0 +1,12 @@
+/* Copyright 2018 Onestein
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
+
+.o_existing_attachment_cell .o_existing_attachment_size {
+    position: absolute;
+    padding: 3px;
+    background-color: rgba(0, 0, 0, 0.8);
+    color: #FFF;
+    border-radius: 0 0 0 2px;
+    cursor: pointer;
+    font-size: 80%;
```

Comment:
It's way better if you use a less file and use Bootstrap variables for such properties, or even better to use a normal bootstrap class such as: https://getbootstrap.com/docs/3.3/components/#labels

Reply:
Yes, but I would be changing the label's style to match the other buttons.

Reply:
Well, the point is that if you leverage Bootstrap, you make your thing look good no matter the theme; if you don't, i.e. a dark theme would render this ugly.

Non-blocking in any case IMHO

Reply:
Yes, I understand it.